projects/media-viewer/src/lib/toolbar/main-toolbar/left-pane/left-pane.component.ts

```diff
@@ -9,26 +7,28 @@ import { ToolbarButtonVisibilityService } from '../../toolbar-button-visibility.
   templateUrl: './left-pane.component.html',
   styleUrls: ['../../../styles/main.scss']
 })
-export class ToolbarLeftPaneComponent {
-  // Input Properties
-  @Input() changePageByDelta: Subject<ChangePageByDeltaOperation>;
-  @Input() setCurrentPage: Subject<SetCurrentPageOperation>;
-  @Input() pageNumber = 1;
+export class ToolbarLeftPaneComponent implements OnInit {
+
+  public pageNumber = 1;
 
   constructor(
-    public readonly toolbarEventsService: ToolbarEventService,
+    public readonly toolbarEvents: ToolbarEventService,
     public readonly toolbarButtons: ToolbarButtonVisibilityService
   ) {}
 
+  public ngOnInit(): void {
+    this.toolbarEvents.setCurrentPage.subscribe(pageNumber => this.setCurrentPage(pageNumber));
```

Comment:
cool, but don’t forget to unsubscribe in ngOnDestroy

suggestion:
    ngOnDestroy(): void {
    // Clean up any subscriptions that we may have
    for (const subscription of this.subscriptions) {
      subscription.unsubscribe();
    }
  }

Reply:
done